Polar/inputs-preprocess/upload-to-s3.py

```diff
@@ -0,0 +1,22 @@
+from xcube.core.store import find_data_store_extensions
```

Comment:
I have adjusted the path while testing to the output in order not to overwrite your work, @davemlz

This script fails with:

`15:23 $ python inputs-preprocess/upload-to-s3.py
````
Traceback (most recent call last):
  File "/home/alicebalfanz/cube-gen/Polar/inputs-preprocess/upload-to-s3.py", line 15, in <module>
    cubes = [glob(f"{cube}/*100m.zarr")[0] for cube in cubes]
  File "/home/alicebalfanz/cube-gen/Polar/inputs-preprocess/upload-to-s3.py", line 15, in <listcomp>
    cubes = [glob(f"{cube}/*100m.zarr")[0] for cube in cubes]
IndexError: list index out of range
```